### ISSUE TITLE
groupby on single column fails

### ISSUE THREAD

**Issue by gtklocker** [opensource9ja/danfojs#142]
**Describe the bug**
Running the example code from [this page](https://danfo.jsdata.org/api-reference/groupby/groupby.cumsum) results in an error.

**To Reproduce**
1. Create a minimal html file with these contents:
```html
<!doctype html>
<html>
  <head>
    <script src="https://cdn.jsdelivr.net/npm/danfojs@0.2.3/lib/bundle.min.js"></script>
  </head>
  <body>
    <script>
      let data ={'A': ['foo', 'bar', 'foo', 'bar',
                      'foo', 'bar', 'foo', 'foo'],
                 'B': ['one', 'one', 'two', 'three',
                      'two', 'two', 'one', 'three'],
                 'C': [1,3,2,4,5,2,6,7],
                 'D': [3,2,4,1,5,6,7,8]
              }
      
      let df = new dfd.DataFrame(data)
      
      let grp = df.groupby(["A"])
      grp.col(["C"]).cumsum().head().print()
      grp.col(["C"]).cumsum().tail().print()
    </script>
  </body>
</html>
```
2. Open in a browser
3. Open the console log
4. See error:
Uncaught TypeError: Cannot read property 'undefined' of undefined
    at eval (eval at arithemetic (bundle.min.js:6), <anonymous>:1:21)
    at GroupBy.arithemetic (bundle.min.js:6)
    at GroupBy.value (bundle.min.js:6)
    at test.html:19

**Expected behavior**
No error.

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser: Chrome
 - Version: 88.0.4324.192

**Comment by risenW** [comment]
@gtklocker Thanks for opening this. I was able to reproduce it, I see it throws that row when grouping by a single column.

**Comment by risenW** [comment]
> @gtklocker Thanks for opening this. I was able to reproduce it, I see it throws that row when grouping by a single column. @steveoni will work on it.

**Comment by gtklocker** [comment]
fwiw I get a similar but different error with `.sum`, but haven't had time to look into it

**Comment by steveoni** [comment]
@gtklocker now fixed in #150
